app/controllers/sites_controller.rb

```diff
@@ -1,5 +1,17 @@
 class SitesController < ApplicationController
   before_action :set_site
-  load_and_authorize_resource
+  load_and_authorize_resource instance_variable: :site, class: 'Site' # descendents still auth Site
   layout 'dashboard'
+
+  def update
```

Comment:
It is a little strange, but not much weirder than having a model base class wrapping/overriding ActiveRecord methods, or all of ActiveFedora.  The inheriting classes here all override, so as a pragmatic thing, it doesn't matter.  I could imagine taking the base class lower and having the controllers (including this one) inherit it, so that no `#update` was inherited.  But the idea that mis-inheriting will prove weird is a hypothetical.  I am satisfied to defer the refactor until such time as we have another descendant that doesn't override `#update` (doesn't seem likely/imminent).